processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -511,6 +522,19 @@ public boolean isStatic() {
         return executable.getModifiers().contains( Modifier.STATIC );
     }
 
+    @Override
+    public boolean isDefault() {
+        boolean isDefault = false;
+        if ( mapperToImplement != null ) {
```

Comment:
But those could be default methods, too? I guess it's mood if you process the other comment and not consider the type at all.